tutorial/pyam_first_steps.ipynb

```diff
@@ -87,7 +87,7 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "import pyam_analysis as iam\n",
+    "import pyam as iam\n",
```

Comment:
suggest to drop the `import as iam` notation, which I introduced mainly because `pyam-analysis` was too long. I can change that throughout the tutorial notebook later when I send the pull request updating it (making sure that the description text is in line with the content of the cells).